lib/sass/script/value/map.rb

```diff
@@ -43,6 +43,50 @@ def eq(other)
       Bool.new(other.is_a?(Map) && value == other.value)
     end
 
+    # Walks the Map, directed by the list
```

Comment:
These days, we wrap comments at 80 columns. Older functions don't do this, but new ones should.